lib/cylc/network/server.py

```diff
@@ -1218,3 +1225,23 @@ def trigger_tasks(self, task_globs, back_out=False):
         self.schd.command_queue.put(
             ("trigger_tasks", (task_globs,), {"back_out": back_out}))
         return (True, 'Command queued')
+
+    # UIServer Data Commands
+    #
+    @authorise(Priv.IDENTITY)
+    @ZMQServer.expose
+    def pb_entire_workflow(self):
+        """Return information about this API.
+
+        Returns a list of callable endpoints.
+
+        Args:
+            encode (bool, optional):
```

Comment:
Missing arg in the function? Was the docstrings of this function copied from another function and need updating perhaps?

Reply:
Yes, Thanks! I did have an encode argument, but decided to always provide encoded.